### PR TITLE
Add resource policies to RNC & RCgNMI app

### PR DESCRIPTION
**Pull request by PeterSuna**
Add resource limits to Helm lighty application

Solve `resource_policies` reported in issues:  #1048 #1047 

ID: 1842

### CODE REVIEW

File: lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app-helm/helm/lighty-rnc-app-helm/values.yaml [hunk 1]
```diff
@@ -88,14 +88,10 @@ deploymentSecurity:
...
-  # choice for the user. This also increases chances charts run on environments with little
-  # resources, such as Minikube. If you do want to specify resources, uncomment the following
-  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-  # limits:
-  #   memory: 512Mi
-  #   cpu: 1000m
-  # requests:
-  #   memory: 256Mi
-  #   cpu: 500m
+resources:
```
Comment (ihrasko) [inline lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app-helm/helm/lighty-rnc-app-helm/values.yaml:91]: Too high request and too low limit.

File: lighty-applications/lighty-rcgnmi-app-aggregator/lighty-rcgnmi-app-helm/helm/lighty-rcgnmi-app-helm/values.yaml [hunk 2]
```diff
@@ -65,14 +65,10 @@ deploymentSecurity:
...
-  # choice for the user. This also increases chances charts run on environments with little
-  # resources, such as Minikube. If you do want to specify resources, uncomment the following
-  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-  # limits:
-  #   memory: 512Mi
-  #   cpu: 1000m
-  # requests:
-  #   memory: 256Mi
-#   cpu: 500m
+resources:
```
Comment (ihrasko) [inline lighty-applications/lighty-rcgnmi-app-aggregator/lighty-rcgnmi-app-helm/helm/lighty-rcgnmi-app-helm/values.yaml:68]: Too high request and too low limit.

**Review by ihrasko — CHANGES_REQUESTED** [review]

File: lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app-helm/helm/lighty-rnc-app-helm/values.yaml (hunk 1 above)
Comment (PeterSuna) [reply lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app-helm/helm/lighty-rnc-app-helm/values.yaml:91]: I set it double as default values was, due to application failure. With this resource limits, the application was running successfully, and I was able to connect the device. This resource values also can be override in helm deployment command.

If you have in mind some specific values, please write it here.

Comment (PeterSuna) [reply lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app-helm/helm/lighty-rnc-app-helm/values.yaml:91]: Done

File: lighty-applications/lighty-rcgnmi-app-aggregator/lighty-rcgnmi-app-helm/helm/lighty-rcgnmi-app-helm/values.yaml (hunk 2 above)
Comment (PeterSuna) [reply lighty-applications/lighty-rcgnmi-app-aggregator/lighty-rcgnmi-app-helm/helm/lighty-rcgnmi-app-helm/values.yaml:68]: Done

**Review by ihrasko — APPROVED** [review]

**Pull request merged**